fs/inode/fs_inodefind.c

```diff
@@ -55,12 +55,7 @@ int inode_find(FAR struct inode_search_s *desc)
    * references on the node.
    */
 
-  ret = inode_lock();
-  if (ret < 0)
-    {
-      return ret;
-    }
-
+  inode_lock();
```

Comment:
inode_rlock

Reply:
let's create a patch to change rwsem which promote rlock to wlock if the same thread already hold wlock.

Reply:
https://github.com/apache/nuttx/pull/13797 Done~